### PR TITLE
[Bug] Set Speaker Audio Device as the default option

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
## Purpose

* Set Speaker Audio Device as the default option

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Fresh install the app
*  Tap on `Start Experience` 
*  Observe whether the default audio device is Speaker
*  Connect to a bluetooth or headphone, then disconnect it
*  Observe whether it get back to Speaker instead of receiver
*  Tap on `Join call`
*  Observe whether the default audio device is Speaker
*  Connect to a bluetooth or headphone, then disconnect it
*  Observe whether it get back to Speaker instead of receiver

### CODE REVIEW

**Review by emlynmac — DISMISSED** [review]
LGTM; just one question about doing this; is there a risk of feedback at call start with speaker / microphone active?

**Comment by Leah-Xia-Microsoft** [comment]
> LGTM; just one question about doing this; is there a risk of feedback at call start with speaker / microphone active?

Good point. I've tested on 2 physical devices joining the call at the same time with speaker / microphone active. Seems not causing feedback.

**Review by emlynmac — APPROVED** [review]

File: AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/Manager/AudioSessionManager.swift [hunk 1]
```diff
@@ -123,12 +123,12 @@ class AudioSessionManager: AudioSessionManagerProtocol {
             case .headphones, .headsetMic:
                 return .headphones
             case .builtInSpeaker:
```
Comment (kevinyulu) [inline AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/Manager/AudioSessionManager.swift:125]: should this be changed to .buildInReceiver?

Comment (Leah-Xia-Microsoft) [reply AzureCommunicationUI/sdk/AzureCommunicationUICalling/Sources/Presentation/Manager/AudioSessionManager.swift:125]: Good catch. It also make me doubt about the scenario that after disconnect to bluetooth, do we want to fall back to receiver or speaker. I'll have to confirm with Alex on this when he's back to the office. Closing this PR for now. Thanks.

**Pull request closed without merging**